app/models/line_item.rb

```diff
@@ -22,4 +22,8 @@ class LineItem < ApplicationRecord
   def value_per_line_item
     item.value_in_cents * quantity
   end
+
+  def package_count
+    quantity / item.package_size.to_f if item.package_size
```

Comment:
Should this be an integer? I don't see how we can have half of a package..rs

Reply:
The screenshot uses an example, e.g. the package size is 10 and 5 are distributed, so you have half a package. But if you feel it makes more sense we could use an integer instead. Would we round up or down in the case of partial packages? @marclerodrigues

Reply:
@seanmarcia what do you think?

Reply:
I just checked with several of the diaper banks and they'd prefer the decimal point in there. Some because it will show them that htey made a mistake and others  will give 1/2 bags (for example.) I imagine we'll want to not display it as 36.0 when there are 36 packages -- I think we can use `sprintf("%g", package_count)` or something like that to address that need! Other than this minor thing this looks great!!

Reply:
I don't think we can use sprintf like I thought. It will choke on nisl I think we could also do something ugly like:
```
quantity = quantity / item.package_size.to_f if item.package_size
quantity = quantity.to_i if (quantity.to_i == quantity) && quantity
quantity
```
Maybe anyway? I'm sure you can come up with something much more clever :)

Reply:
`sprintf("%g", package_count || 0)` might also work?

```ruby
2.6.2 :005 > sprintf("%g", 1.5 || 0)
 => "1.5" 
2.6.2 :006 > sprintf("%g", 2 || 0)
 => "2" 
2.6.2 :007 > sprintf("%g", nil || 0)
 => "0" 
2.6.2 :008 > sprintf("%g", 0 || 0)
 => "0" 
```

Reply:
I thought about that. Then we're going to show 0 rather than nothing. I think the preferred outcome would be to leave it blank on the pdf. Especially if they use package size for some and not others.

Reply:
Technically, `0` is a correct response because the package_count is 0.

If we want it to display nothing in the view, then we can handle that in the view layer?

Like -- have this method return 0, but then in the view, something like: 

```ruby
<%= pluralize('package', line_item.package_count) unless line_item.package_count.zero? %>
```